devel/api.md

```diff
@@ -655,6 +661,57 @@ Type: `string`
 
 The interface name which current VLAN is based on.
 
+# VXLAN
+Besides basic interface properties, each VXLAN interface state also contains
+a dictionary saved in key `VXLAN.CONFIG_SUBTREE`.
+
+Example:
+
+```python
+{
+    Interface.KEY: [
+        Interface.NAME: "eth1.101",
+        Interface.TYPE: VXLAN.TYPE,
+        Interface.STATE: InterfaceState.UP
+        VXLAN.CONFIG_SUBTREE: {
+            VXLAN.ID: 101,
+            VXLAN.BASE_IFACE: 'eth1'
+            VXLAN.REMOTE: '192.168.3.3'
+        }
+    ]
+}
+```
+
+## `VXLAN.ID`
+
+Type: `integer`
+
+The VXLAN ID.
+
+To change VXLAN ID, please use `InterfaceState.ABSENT` for existing one and add
```

Comment:
Proposal: To change *the* VXLAN ID,  please remove the interface first by setting the state to `InterfaceState.ABSENT` and then add a new one.